src/pineko/evolve.py

```diff
@@ -172,21 +190,37 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
             "are you sure that's what you want?"
         )
 
-    with open(card_path, "w", encoding="UTF-8") as f:
-        yaml.safe_dump(operators_card, f)
-        f.write(f"# {pineko_version=}")
+    def dump_card(card_path, operators_card, conv_type):
+        operators_card["configs"]["polarized"] = conv_type == "polPDF"
```

Comment:
this is actually a bit dangerous since you are modifying in place ...

Reply:
yes, indeed I can deepcopy.